dtm0/service.c

```diff
@@ -223,16 +268,35 @@ static void dtm0_service_fini(struct m0_reqh_service *service)
         m0_free(service);
 }
 
-int m0_dtm0_stype_init(void)
+M0_INTERNAL int m0_dtm0_stype_init(void)
 {
 	return m0_reqh_service_type_register(&dtm0_service_type);
 }
 
-void m0_dtm0_stype_fini(void)
+M0_INTERNAL void m0_dtm0_stype_fini(void)
 {
 	m0_reqh_service_type_unregister(&dtm0_service_type);
 }
 
+static enum m0_dtm0_service_origin origin(struct m0_reqh_service *service)
+{
+	return container_of(service,
+			    struct m0_dtm0_service,
+			    dos_generic)->dos_origin;
+}
+
+M0_INTERNAL bool m0_dtm0_is_a_volatile_dtm(struct m0_reqh_service *service)
+{
+	return service->rs_type->rst_typecode == M0_CST_DTM0 &&
+		origin(service) == DTM0_ON_VOLATILE;
+}
+
+M0_INTERNAL bool m0_dtm0_is_a_persistent_dtm(struct m0_reqh_service *service)
+{
+		return service->rs_type->rst_typecode == M0_CST_DTM0 &&
```

Comment:
Indentation level seems to be wrong here. One '\t' is enough.

Reply:
fixed.